### ISSUE TITLE
writing in "code" folder

### ISSUE THREAD

**Issue by mikedecr** [Peter-Erickson/polisci-904#1]
Just a heads up, it isn't conventional have store your writing in a "code" folder. Typically that exists in the project root or in a dedicated folder for writing. "Code" would be for data analysis scripts that run separately from the written output!

**Comment by Peter-Erickson** [comment]
Mike,

Question for you on this: did this happen because I created the local
repository too early in my file working directory?

Respectfully,

Pete Erickson

**Comment by mikedecr** [comment]
Nope, this is just your choice of which folder to keep those files. You
could simply move them into a "writing" folder (as long as it's still in
your project repo) and then push the changes and it will be all same-same.

Michael DeCrescenzo
Ph.D. Candidate, Political Science
University of Wisconsin–Madison
decrescenzo@wisc.edu
